examples/step-14/doc/results.dox

```diff
@@ -47,56 +47,62 @@ Refinement cycle: 8
 @endcode
 
 
-First let's look what the program actually computed. On the fifth
-grid, primal and dual numerical solutions look like this:
+First let's look what the program actually computed. On the seventh
+grid, primal and dual numerical solutions look like this (using a
+color scheme intended to evoke the snow-capped mountains of
+Colorado that the original author of this program now calls
+home):
 <table align="center">
   <tr>
     <td width="50%">
-      <img src="https://www.dealii.org/images/steps/developer/step-14.point-value.solution-5.png" alt="">
+      <img src="https://www.dealii.org/images/steps/developer/step-14.point-value.solution-7.9.2.png" alt="">
     </td>
     <td width="50%">
-      <img src="https://www.dealii.org/images/steps/developer/step-14.point-value.solution-5-dual.png" alt="">
+      <img src="https://www.dealii.org/images/steps/developer/step-14.point-value.solution-7-dual.9.2.png" alt="">
     </td>
   </tr>
 </table>
-Obviously, the region at the bottom left is so unimportant for the
+Apparently, the region at the bottom left is so unimportant for the
 point value evaluation at the top right that the grid is left entirely
-unrefined there, even though the solution has singularities there! Due
+unrefined there, even though the solution has singularities at the inner
+corner of that cell! Due
 to the symmetry in right hand side and domain, the solution should
 actually look like at the top right in all four corners, but the mesh
 refinement criterion involving the dual solution chose to refine them
-differently.
+differently -- because we said that we really only care about a single
+function value somewhere at the top right.
 
 
 
 Looking at the grids that are produced in the course of subsequent
-refinement, here are some of them:
+refinement, here are some of them (in refinement cycles 0, 2, 4 in
```

Comment:
I think the original sentence actually doesn't sound right when I read it. What about "Here are some of the grids that are produced in the course of subsequent refinement"